Source/scrollrt.cpp

```diff
@@ -781,10 +781,10 @@ static void scrollrt_draw_dungeon(CelOutputBuffer out, int sx, int sy, int dx, i
 			} else {
 				Cl2DrawLight(out, px, dy, pCelBuff, nCel, pDeadGuy->_deadWidth);
 			}
-		} while (0);
+		} while (false);

```

Comment:
A do {...} while(false); loop? Someone tried very hard to avoid a goto.

Reply:
`goto` is generally not very safe in C++.